server/liveblog/themes/themes.py

```diff
@@ -239,12 +238,20 @@ def publish_related_blogs(self, theme):
         terms = []
         for t in self.get_children(theme['name']) + [theme['name']]:
             terms.append({'term': {'blog_preferences.theme': t}})
```

Comment:
This is not needed any more I guess. But you can comment it just after the `FIXME`